toolchain.txt

```diff
@@ -18,7 +18,8 @@ has_function_printf = false
 c_args = [
   '-mabi=ilp32',
   '-march=rv32imc',
-  '-mcmodel=medany']
+  '-mcmodel=medany',
+]
```

Comment:
Your build error is because of this change. I tried to write it this way too, but the config parser in meson/python cares too deeply about whitespace.

Reply:
Ugh.